File: C/1010.c
#include <stdio.h>

int main()
{
    int item, qty1;
    float price1;
    scanf("%d %d %f", &item, &qty1, &price1);
    int qty2;
    float price2;
    scanf("%d %d %f", &item, &qty2, &price2);
    float total = (qty1 * price1) + (qty2 * price2);
    printf("VALOR A PAGAR: R$ %.2f\n", total);
    return 0;
}

File: Dart/1010.dart
import 'dart:io';

void main() {
  String input1 = stdin.readLineSync();
  String input2 = stdin.readLineSync();
  int qty1 = int.parse(input1.split(' ')[1]);
  double price1 = double.parse(input1.split(' ')[2]);
  int qty2 = int.parse(input2.split(' ')[1]);
  double price2 = double.parse(input2.split(' ')[2]);
  double total = (qty1 * price1) + (qty2 * price2);
  print("VALOR A PAGAR: R\$ " + total.toStringAsFixed(2));
}
